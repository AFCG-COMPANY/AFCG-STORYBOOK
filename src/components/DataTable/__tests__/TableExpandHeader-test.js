import React from 'react';
import { mount } from 'enzyme';
import { Table, TableHead, TableRow, TableExpandHeader } from '../';

describe('DataTable.TableExpandHeader', () => {
  it('should render', () => {
    const wrapper = mount(
      <Table>
        <TableHead>
          <TableRow>
            <TableExpandHeader className="custom-class" />
          </TableRow>
        </TableHead>
      </Table>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
