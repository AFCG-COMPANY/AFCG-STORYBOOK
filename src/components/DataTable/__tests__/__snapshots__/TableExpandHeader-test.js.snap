// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable.TableExpandHeader should render 1`] = `
<Table>
  <table
    className="bx--data-table-v2 bx--data-table-v2--zebra"
  >
    <TableHead>
      <thead>
        <TableRow>
          <tr>
            <TableExpandHeader
              className="custom-class"
            >
              <th
                className="custom-class"
                scope="col"
              />
            </TableExpandHeader>
          </tr>
        </TableRow>
      </thead>
    </TableHead>
  </table>
</Table>
`;
